nvm.sh

```diff
@@ -221,6 +228,10 @@ nvm_clang_version() {
   clang --version | command awk '{ if ($2 == "version") print $3; else if ($3 == "version") print $4 }' | command sed 's/-.*$//g'
 }
 
+nvm_curl_version() {
+  curl --version | command awk '{ if ($1 == "curl") print $2 }' | command sed 's/-.*$//g'
```

Comment:
`nvm_curl_libz_support` uses `curl -V`; are those identical across all curl versions?

If you're not certain, let's stick with `curl -V` to be consistent.